libs/admin/src/lib/components/user-edit/user-edit.component.html

```diff
@@ -1,44 +1,103 @@
-<form [formGroup]="formGroup">
-  <div class="edit-area">
+<div class="user-edit">
+  <form [formGroup]="formGroup">
     <div *ngIf="user" class="header">
-      <div class="email">
+      <h2 class="email">
         {{ user.email }}
+      </h2>
+    </div>
+    <div class="user-edit__user-data">
+      <div class="user-edit__user-data__group">
+        <h3 translate>admin.userPage.lang</h3>
+        <mat-form-field class="lang-form-field">
+          <mat-select formControlName="lang">
+            <mat-option value="de" translate><span translate>admin.languages.de</span></mat-option>
+            <mat-option value="en" translate><span translate>admin.languages.en</span></mat-option>
+            <mat-option value="fr" translate><span translate>admin.languages.fr</span></mat-option>
+            <mat-option value="it" translate><span translate>admin.languages.it</span></mat-option>
+            <mat-option value="rm" translate><span translate>admin.languages.rm</span></mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+      <div class="user-edit__user-data__group">
+        <h3 translate>admin.userPage.admin</h3>
+        <mat-checkbox formControlName="isAdmin" color="primary"></mat-checkbox>
       </div>
     </div>
-    <div class="label-heading label-heading--before-form-field">Sprache</div>
-    <mat-form-field class="lang-form-field">
-      <mat-label>Sprache</mat-label>
-      <mat-select formControlName="lang">
-        <mat-option value="de">DE</mat-option>
-        <mat-option value="en">EN</mat-option>
-        <mat-option value="fr">FR</mat-option>
-        <mat-option value="it">IT</mat-option>
-        <mat-option value="rm">RM</mat-option>
-      </mat-select>
-    </mat-form-field>
+    <div class="user-edit__workgroups">
+      <div class="user-edit__workgroups__header">
+        <h2 translate>admin.workgroups</h2>
+        <form>
```

Comment:
Already included workgroups can be selected again. They should be disabled instead.
`admin.name` is visible in the component.
After workgroup selection, the input should be empty again.